app/controllers/posts_controller.rb

```diff
@@ -50,6 +50,21 @@ def interactions
     end
   end
 
+  def mentionable_in_comment
+    respond_to do |format|
+      format.json {
+        if params[:id].present? && params[:q].present?
```

Comment:
>  don't understand what you mean here? There will be an @. So you type the @ and you already see the last few commenters and the post author as suggestion, but without the need to type the actual name (so you can select them with arrow-keys). I don't see where this should be "counter intuitive"?

Ah, misunderstanding. My bad, sry.

> That's wrong. The data you need is the post author and the comment authors. And this information is already available in the front end.

Where is it available? If so, it's easilly doable, then, yes.

> That's wrong. The data you need is the post author and the comment authors. And this information is already available in the front end.

As stated before, I can take care of the front-end.